examples/cis2-multi/src/lib.rs

```diff
@@ -894,13 +1399,17 @@ fn contract_permit(
 /// Logs a `UpdateOperator` event. The function assumes that the sender is
 /// authorized to do the `updateOperator` action.
 fn update_operator(
+    paused: bool,
```

Comment:
**Suggestion (non-blocking):** The state is already passed as an argument, so we could just read `paused` from there.